### PR TITLE
Mongo/himanshu

### PR DESCRIPTION
**Pull request by HGoyal09**
Merging this pr will - 
Extract data from the mongo server

### CODE REVIEW

File: server/dal/stock-info.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+var Schema = mongoose.Schema
+
+var stockInfoSchema = new Schema({
+    company_name: String,
+    ticker: String,
+    current_price: Number,
+    previous_close: Number,
+    open: Number,
+    last_price: Number,
```
Comment (paulsgh) [inline server/dal/stock-info.js:15]: I believe one of the decisions during yesterday's call was to remove `last_price`

File: server/dal/stock-info.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const dotenv = require('dotenv');
+dotenv.config();
+
+mongoose.connect(process.env.MONGO_DB_CONNECTION_STRING, { useNewUrlParser: true, useUnifiedTopology: true })
+
+var Schema = mongoose.Schema
+
+var stockInfoSchema = new Schema({
+    company_name: String,
```
Comment (paulsgh) [inline server/dal/stock-info.js:10]: During yesterday's call we also discussed how `company_name` and `description` (which isn't listed in this schema) will be constants; `ticker` too of course. The rest of these values won't be.

Not sure if that means we should remove the `company_name` and `ticker` from this schema.

**Review by paulsgh — DISMISSED** [review]

File: server/dal/stock-info.js (hunk 1 above)
Comment (HGoyal09) [reply server/dal/stock-info.js:15]: Dw about this 
This was a basic working setup of mongodb 
The schemas will change

File: server/dal/stock-info.js (hunk 2 above)
Comment (HGoyal09) [reply server/dal/stock-info.js:10]: Same as above

**Review by weiyoung — APPROVED** [review]
LGTM

**Pull request merged**